src/engines/aurora/renderwalkmesh.cpp

```diff
@@ -0,0 +1,89 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  Renders wrapped walkmesh for debug purposes.
+ */
+
+#include "src/engines/aurora/renderwalkmesh.h"
+
+namespace Engines {
+
+RenderableWalkmesh::RenderableWalkmesh()
+		: Renderable(Graphics::kRenderableTypeObject), _dirty(true) {
```

Comment:
I'd prefer it if the parent class initialization was in the first line.

I.e
```
RenderableWalkmesh::RenderableWalkmesh() : Renderable(Graphics::kRenderableTypeObject),
	_dirty(true) {
````

That's how most of the existing code does it, especially when the line isn't too long.